cmd/root.go

```diff
@@ -0,0 +1,145 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/adrg/xdg"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"sigs.k8s.io/hydrophone/pkg/client"
+	"sigs.k8s.io/hydrophone/pkg/common"
+	"sigs.k8s.io/hydrophone/pkg/service"
+)
+
+var (
+	cfgFile          string
+	kubeconfig       string
+	parallel         int
+	verbosity        int
+	outputDir        string
+	cleanup          bool
+	listImages       bool
+	conformance      bool
+	focus            string
+	skip             string
+	conformanceImage string
+	busyboxImage     string
+	dryRun           bool
+	testRepoList     string
+	testRepo         string
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "hydrohpone",
+	Short: "Hydrophone is a lightweight runner for kubernetes tests.",
+	Long:  `Hydrophone is a lightweight runner for kubernetes tests.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		client := client.NewClient()
+		config, clientSet := service.Init(viper.GetString("kubeconfig"))
+		client.ClientSet = clientSet
+		common.PrintInfo(client.ClientSet, config)
+		if cleanup {
+			service.Cleanup(client.ClientSet)
+		} else if listImages {
+			service.PrintListImages(client.ClientSet)
+		} else {
+			common.ValidateArgs(client.ClientSet, config)
+
+			service.RunE2E(client.ClientSet)
+			client.PrintE2ELogs()
+			client.FetchFiles(config, clientSet, viper.GetString("output-dir"))
+			client.FetchExitCode()
+			service.Cleanup(client.ClientSet)
+		}
+		log.Println("Exiting with code: ", client.ExitCode)
+		os.Exit(client.ExitCode)
+	},
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		log.Println(err)
```

Comment:
you can use [log.Fatal](https://github.com/kubernetes-sigs/hydrophone/blob/main/pkg/log/logs.go#L42) or log.Fatalf instead  which prints the error and exits

Reply:
Fixed, thanks